app/src/main/res/layout/upload_picture.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
the name convention for layout is to begin with activity if it is an activity or fragment if it is a fragment. So this should be activity_upload_picture